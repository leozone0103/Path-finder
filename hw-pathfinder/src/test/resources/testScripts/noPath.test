# Test when no path exists
CreateGraph graph
AddNode graph A
AddNode graph B
FindPath graph A B