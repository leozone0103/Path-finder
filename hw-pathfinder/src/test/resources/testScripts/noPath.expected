# Test when no path exists
created graph graph
added node A to graph
added node B to graph
path from A to B: